caikit_tgis_backend/tgis_connection.py

```diff
@@ -94,6 +95,7 @@ def from_config(cls, model_id: str, config: dict) -> Optional["TGISConnection"]:
         """Create an instance from a connection template and a model_id"""
         hostname = config.get(cls.HOSTNAME_KEY)
         if hostname:
+            # TODO: Namespace needs to be populated here as well into the hostname template
```

Comment:
The hostname needs to end up being the FQDN which includes the namespace.
As a result this template mapping step needs to support an additional `namespace` field to be read from the config.

This also requires an additional `HOSTNAME_TEMPALTE_NAMESPACE` class constant to identify the config key.